client/ayon_core/tools/push_to_project/main.py

```diff
@@ -4,14 +4,20 @@
 from ayon_core.tools.push_to_project.ui import PushToContextSelectWindow
 
 
-def main_show(project_name, version_id):
-    app = get_ayon_qt_app()
-
-    window = PushToContextSelectWindow()
+def main_show(project_name, version_id, library_filter, context_only):
```

Comment:
Why is needed library filter?

Reply:
By default the tool only displays library projects, but we need to show all projects.